### PR TITLE
ci: fix GitHub Token

### PR DESCRIPTION
**Pull request by fharper**

### CODE REVIEW

**Comment by fharper** [comment]
Link check won't pass until this PR is merged https://github.com/kubefirst/docs/pull/372 and main is merged in this PR after, but you can still approve it, merging from main won't require another approval.

**Review by claywd — APPROVED** [review]

**Pull request merged**
